modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -13483,6 +13484,9 @@ set-the-google-analytics-id-that-is-used-for-this-set-of-pages=Set the Google An
 set-the-google-maps-api-key-that-is-used-for-this-set-of-pages=Set the Google Maps API Key that is used for this set of pages.
 set-the-google-places-api-key-that-is-used-for-this-set-of-pages=Set the Google Places API Key that is used for this set of pages.
 set-the-key-for-the-bing-maps-api-integration=Set the key for the Bing Maps API integration. See https://msdn.microsoft.com/en-us/library/ff428642.aspx for more information.
+set-the-maximum-number-of-characters=Set the Maximum Number of Characters
+set-the-maximum-number-of-characters-accepted-this-value-cant-be-less-than-1-or-greater-than-280=Set the maximum number of characters accepted. This value cannot be less than 1 or greater than 280.
```

Comment:
Hey, @rebsilva could these two be only one with a parametrized max and min?

Reply:
Sure @rodrigopaulino! I will try to do that, thanks!